package.json

```diff
@@ -85,7 +85,7 @@
     "@netlify/plugin-edge-handlers": "^3.0.0",
     "@netlify/plugins-list": "^6.2.0",
     "@netlify/routing-local-proxy": "^0.34.1",
-    "@netlify/zip-it-and-ship-it": "5.3.1",
+    "@netlify/zip-it-and-ship-it": "5.4.0",
```

Comment:
Shouldn't this be `^5.4.0`?

Reply:
I think this doesn't make a difference, given that we're shipping a `npm-shrinkwrap.json` file.